src/OpenSage.Game/Data/Utilities/ParseUtility.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Globalization;
+
+namespace OpenSage.Data.Utilities
+{
+    internal static class ParseUtility
+    {
+        public static float ParseFloat(string s)
+        {
+            return float.Parse(s, CultureInfo.InvariantCulture);
+        }
+
+        public static bool TryParseFloat(string s, out float result)
+        {
+            return float.TryParse(s, out result);
```

Comment:
Shouldn't this use the `TryParse` overload that you can pass a culture to?

Reply:
Oops.